openbsd-compat/bsd-misc.c

```diff
@@ -143,7 +143,11 @@ utimensat(int fd, const char *path, const struct timespec times[2],
 		return -1;
 	}
 # ifndef HAVE_FUTIMES
+#ifndef __TANDEM
 	return utimes(path, tv);
```

Comment:
what's the reason for this change?  const again?

Reply:
Yes

Reply:
Specifically const misinterpretation by the compiler.

Reply:
I'm not going to carry a diff like this to paper over a a compiler bug, especially for such an esoteric platform.  If you want to put the equivalent '-Dconst' in CFLAGS in the tandem block that's fine.